lib/configure.js

```diff
@@ -1,6 +1,7 @@
 'use strict'
 
 const { promises: fs } = require('graceful-fs')
```

Comment:
```suggestion
const { promises: fs, readFileSync } = require('graceful-fs')
```

Reply:
I find this easier to refactor in the future especially since we have to mock the individual methods for tests.